app/_data/docs_nav_kuma_dev.yml

```diff
@@ -126,6 +126,22 @@ items:
             url: /production/upgrades-tuning/fine-tuning/
       - text: Init containers
         url: /production/init-containers/
+  - title: Using Kong Mesh
```

Comment:
Shouldn't it be `Kuma`?

Reply:
we have a placeholder - `{{site.mesh_product_name}}` - would that work? if that's not the case maybe then: "using the product"?